embedly/embedly/api/views.py

```diff
@@ -43,3 +43,14 @@ def extract_urls():
         status=200,
         mimetype='application/json',
     )
+
+
+@blueprint.route('/v2/extract', methods=['POST'])
+def extract_urls_v2():
+    urls = json.loads(request.values.get('urls', '[]'))
```

Comment:
@oyiptong @k88hudson Error catching and reporting is handled in #22 where all of this feedback will be accounted for!